### PR TITLE
Added basic DB implementation

### LINKED ISSUE #6: Implement DB interface
Add ability to use DB for server

### PR DESCRIPTION
**Pull request by esikachev**
close #6

### CODE REVIEW

File: my_dev_server/db/api.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from my_dev_server.db import base
+
+
+def add_values_to_table(table, dict_of_values):
+    db = base.Connect()
+    insert = table.insert()
+    #clause = db.metadata.tables.slams.insert().values(name='Wimbledon', country='United Kingdom')
```
Comment (IlyaMenkov) [inline my_dev_server/db/api.py]: I think that we do not need this comment, delete this line.

Comment (esikachev) [inline my_dev_server/db/api.py]: @IlyaMenkov totally agree. It will be done in final patch-set

File: my_dev_server/db/models.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+from my_dev_server.db import base
+
+
+class User(base.Base):
+
+    __tablename__ = 'users'
+
+    id = sql.Column(sql.Integer, primary_key=True)
+    username = sql.Column(sql.String(length=15))
```
Comment (zemuvier) [inline my_dev_server/db/models.py]: Let's match length equal to 10

File: my_dev_server/config.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+log.register_options(CONF)
+
+
+def list_opts():
+    return {'DEFAULT': db_opts}
+
+
+def parse_config():
+    CONF(project='my-dev-server', default_config_files=sys.argv[2:])
+    print CONF.db_url
```
Comment (IlyaMenkov) [inline my_dev_server/config.py]: I think this string is not nesessary

**Pull request merged**